crates/ruff/resources/test/fixtures/ruff/RUF017.py

```diff
@@ -0,0 +1,16 @@
+import typing
+from typing import TypeAlias
+
+# RUF017
+x: typing.TypeAlias = int
+x: TypeAlias = int
+
+
+# RUF017 with generics (todo)
+T = typing.TypeVar["T"]
+x: typing.TypeAlias = list[T]
```

Comment:
This case is where the `type` keyword shines but seems much harder to auto-fix?

Reply:
If we could detect that `T` is _only_ used here, would that be sufficient to allow an autofix?

Reply:
Well I'm fine with making T an unused variable and dealing with that later. The part I don't know how to do is recursively find all `TypeVar` bindings used in a complex expression. I'm guessing we can use the visitor somehow or we already have some helpers?

Reply:
I would probably write a custom visitor, like `NameFinder`, that just collects all the `TypeVar` instances.